src/components/lyne-link-button/styles/lyne-link-button.component.scss

```diff
@@ -24,6 +24,7 @@
   @include button();
   text-decoration: none;
   fill: currentColor;
+  width: fit-content;
```

Comment:
Yes, we need to set this, otherwise the button would span 100% if it is part of the stack component. Or do you @mbleuer have an other idea how to solve this?